variables.tf

```diff
@@ -469,3 +469,47 @@ variable "graphdb_enable_userdata_scripts_on_reboot" {
   type        = bool
   default     = false
 }
+
+# KMS CMK
+
+variable "enable_cmk" {
+  description = "Enable CMK for encryption. If false, use AWS managed key."
+  type        = bool
+  default     = false
+}
+
+variable "kms_master_key_id" {
```

Comment:
I don't see this used anywhere in the monitoring module, do we support passing existing keys or not?